cpu/stm32_common/periph/sleepmem.c

```diff
@@ -0,0 +1,59 @@
+#include <string.h>
+
+#include "periph/sleepmem.h"
+
+#include "cpu.h"
+
+#ifndef RTC_BKP_NUMBER
+#define RTC_BKP_NUMBER (20u)
```

Comment:
is this true for all stm32?

Reply:
Nope, from a quick scan L4 has 32, L1 has UP TO 32